src/annotator/integrations/test/html-side-by-side-test.js

```diff
@@ -0,0 +1,92 @@
+import { guessMainContentArea } from '../html-side-by-side';
+
+describe('annotator/integrations/html-side-by-side', () => {
+  let contentElements;
+
+  function createContent(paragraphs) {
+    const paraElements = paragraphs.map(({ content, left, width }) => {
+      const el = document.createElement('p');
+      el.textContent = content;
+      el.style.position = 'absolute';
```

Comment:
Presumably `absolute` per the initial containing block of the `html` element, as no ancestors have an explicit `position` property? Would the behavior of tests change if the containing `div` had a positioning context?

Reply:
Positioning is used as a convenient way to set the X coordinate of the paragraphs. If the containing div was positioned, then the final coordinate would be the left value here plus the X coordinate of the containing `div`.